### PR TITLE
[RV64_DYNAREC] Added more opcodes for Unciv

### PR DESCRIPTION
**Pull request by ksco**

### CODE REVIEW

File: src/dynarec/arm64/dynarec_arm64_f0.c [hunk 1]
```diff
@@ -247,7 +247,7 @@ uintptr_t dynarec64_F0(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                                 CBNZx_MARKLOCK(x4);
                                 // done
                                 MARK;
-                                UFLAG_IF {emit_cmp32(dyn, ninst, rex, x6, x2, x3, x4, x5);}
+                                UFLAG_IF {emit_cmp8(dyn, ninst, rex, x6, x2, x3, x4, x5);}
```
Comment (ptitSeb) [inline src/dynarec/arm64/dynarec_arm64_f0.c:250]: remove `rex` argument, it's not relevent for 8bits operations

File: src/dynarec/rv64/dynarec_rv64_00_1.c [hunk 2]
```diff
@@ -114,7 +114,9 @@ uintptr_t dynarec64_00_1(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
         case 0x66:
             addr = dynarec64_66(dyn, addr, ip, ninst, rex, rep, ok, need_epilog);
             break;
-
+        case 0x67:
+            addr = dynarec64_67(dyn, addr, ip, ninst, rex, rep, ok, need_epilog);
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_00_1.c:118]: Ooh, I forgot the call?!

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_00_1.c:118]: Yep, haha.

File: src/dynarec/arm64/dynarec_arm64_f0.c (hunk 1 above)
Comment (ksco) [reply src/dynarec/arm64/dynarec_arm64_f0.c:250]: Oh, indeed.

**Pull request merged**